src/lib/shim.nim

```diff
@@ -0,0 +1,6 @@
+import json
+import ../lib
+
+proc hashMessage*(message: string): string =
+  let hash = lib.hashMessage(message)
+  $(%*{"result": hash})
```

Comment:
It is possible to avoid doing the json encoding/decoding here by returning a string directly using https://nim-lang.org/docs/strformat.html

Reply:
Ah nice, that should be faster right?

Reply:
Done: 9359715e3b3df47c9e11874e38ec139719a8c482.